src/Jellyfin.Database/Jellyfin.Database.Providers.Sqlite/SqliteDatabaseProvider.cs

```diff
@@ -84,4 +85,36 @@ public void ConfigureConventions(ModelConfigurationBuilder configurationBuilder)
     {
         configurationBuilder.Conventions.Add(_ => new DoNotUseReturningClauseConvention());
     }
+
+    /// <inheritdoc />
+    public Task<string> MigrationBackupFast(CancellationToken cancellationToken)
+    {
+        var key = Guid.NewGuid().ToString("D");
+        var path = Path.Combine(_applicationPaths.DataPath, "jellyfin.db");
```

Comment:
I think it would be good to append the timestamp that the backup was created here
I know that file create/change stamps are a thing but names are also good :)

Reply:
good point, in that case i would exchange the guid with just the timestamp as they _key_ is just arbitrary anyway